components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/inbound/IdentityMessageContext.java

```diff
@@ -37,6 +37,10 @@ public IdentityMessageContext(IdentityRequest request, Map<T1, T2> parameters) {
         this.request = request;
     }
 
+    public IdentityMessageContext(IdentityRequest request) {
+        this.request = request;
+    }
```

Comment:
Fixed after offline discussion. Although still didn't understand how it can be overcome to call the no-arg constructor.